packages/core/src/anchor/anchor-processing-loop.ts

```diff
@@ -46,6 +55,7 @@ export class AnchorProcessingLoop {
           `Anchor event with status ${event.status} for commit CID ${entry.cid} of Stream ${streamId} handled successfully`
         )
         if (isTerminal) {
+          this.#anchorPollingMetrics.record(entry)
```

Comment:
does this internally somehow parse the entry and understand what StreamID it applies to?  Like the age of a request in the polling loop is going to be per-stream.  You might want to take the average across all streams to be what gets exposed, but you still need to track the request lifetime per-stream.  That's the part that's hard to understand just from reading this code.  There's nothing signaling down into the metric system what field from `entry` to use to track the age of the request.  If this *isn't* somehow turning into per-streamid age tracking at the lower level, then this metric likely isn't tracking the right thing.  If you're just recording a timestamp every time this line of code gets hit, you're tracking something weird.  This line will get hit every time *any stream* reaches a terminal state.  So that means generally the loop may run many times without hitting this line, until an anchor batch runs, at which point this line will get many hundreds of times rapid fire, once per each stream anchored in the batch.

Reply:
right, it uses entry.timestamp, the SINCE_FIELD is set on start()

Reply:
okay, I read through the code in https://github.com/ceramicnetwork/observability/pull/16, and now I understand how this is working.  It's getting the start timestamp from the entry itself.  This is super not clear to a reader of the code in this PR though, without a pretty deep understanding of what the service-metrics package is doing internally.  Can we add some comments here to explain?

The good news is now I do think this is actually doing the right thing after all, I just didn't understand it before.